SPECS/glibc/glibc.spec

```diff
@@ -323,6 +322,9 @@ grep "^FAIL: nptl/tst-eintr1" tests.sum >/dev/null && n=$((n+1)) ||:
 %defattr(-,root,root)
 
 %changelog
+* Wed Jan 17 13:56:40 EST 2024 Dan Streetman <ddstreet@ieee.org> - 2.38-2
```

Comment:
can you remove "13:56:40 EST"; most of our changelogs do not have these details

Reply:
ack